src/styles/globals.css

```diff
@@ -2,155 +2,18 @@
 @tailwind components;
 @tailwind utilities;
 
-/* CSS Reset */
```

Comment:
I've removed all of this css reset stuff because we don't need it anymore -- tailwind applies its own reset that does the same job but better.